google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -119,6 +119,7 @@ appengine.deployment.source.not.found.error=Deployment source not found: {0}.
 appengine.deployment.invalid.source.error=Invalid deployment source.
 appengine.deployment.status.deploying=Deploying to App Engine
 appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
+appengine.deployment.deployallconfigs.checkbox.label=Deploy all App Engine service configuration files (indexes, dos, dispatch, cron, queues)
```

Comment:
@etanshaul that's a good point. My wording there is awful.

What about ''Update dos, dispatch, cron, task queues, and datastore indexes" ?